src/main/java/io/github/externschool/planner/service/ScheduleService.java

```diff
@@ -15,6 +15,9 @@
  * @author Benkoff (mailto.benkoff@gmail.com)
  */
 public interface ScheduleService {
+
+    ScheduleEvent saveEvent(ScheduleEvent scheduleEvent);
```

Comment:
there is no need to create this method if you put your new schedule creation method void updateTeacherSchedule() from Teacher Service to this Schedule Service.